content/docs/SUSHI/tips/_index.md

```diff
@@ -144,9 +144,87 @@ resources:
 This will result in your logical model example being listed and displayed as a proper example of the logical model.
 
 {{% alert title="Note" color="success" %}}
-This does not allow/support using the `Instance` keyword for creating examples of logical models. Authors must created the example as raw JSON or XML.  Support for the `Instance` keyword may come in future versions of SUSHI.
+This does not allow/support using the `Instance` keyword for creating examples of logical models. Authors must create the example as raw JSON or XML.  Support for the `Instance` keyword may come in future versions of SUSHI.
 {{% /alert %}}
 
+## Manual Slice Ordering
+
+Starting in SUSHI `v3.0.0`, authors can exercise full manual control over the ordering of slice elements within Instances. Previous versions of SUSHI allowed for partial control of slice element ordering, but some ordering was determined by SUSHI's implementation and could not be affected by an author. In current version of SUSHI (`v3.0.0` or later), authors can configure their FSH projects to manually control slice ordering. When using manual slice ordering, it is recommended to also make use of soft indexing, and to avoid using hard numeric indices.
+
+Manual slice ordering follows the following rules:
+
+* slices appear on an Instance in the order of FSH rules
+* any required slices (`1..*`) that are not referenced in a FSH rule on the Instance appear after all referenced slices in the order in which they are defined on the Instance's StructureDefinition (the instance's `InstanceOf`)
+* a rule that references a sliced element must reference it using the slice name if soft indices are used
+* when rule paths contain soft indices, the resolved numeric index will take into account named slices that have already been referenced by previous rule paths
```

Comment:
And we probably need to be more specific that this pertains to paths w/ soft indices _instead of_ slice names. Maybe something like:
```suggestion
* when rule paths use only a soft index (instead of a slice name), the resolved numeric index will take into account named slices that have already been referenced by previous rule paths. This should only be used to add items that do not belong to a defined slice.
```